src/dynamics/ads_player/ads_player.js

```diff
@@ -116,7 +116,7 @@ Scoped.define("module:Ads.Dynamics.Player", [
                         const dyn = this.parent();
                         if (dyn) {
                             dyn.stopAdsRenderFailTimeout(true);
-                            dyn.channel("ads").trigger("ad-error");
+                            dyn.trigger("ad-error", `Render ads took a too much time`);
```

Comment:
@sambua shouldn't we be sending ad data with this event?
```suggestion
                            dyn.trigger("ad-error", "Ad took too long to render");
```